pkg/controller/controller.go

```diff
@@ -43,8 +45,14 @@ func init() {
 }
 
 func getLogger(opts ...zap.Opts) logr.Logger {
+	// make raw logger object from options in original zap pkg
+	rawOpts := zap.RawZapOpts(ubzap.AddCaller())
+
+	// append k8s zap options to raw options
+	rawLogger := zap.NewRaw(append(opts, rawOpts)...)
+
 	// zap is the default recommended logger for controller-runtime when wanting json structured output
-	return zap.New(opts...)
```

Comment:
we would like to use the controller runtime zap logger. It has some k8s helpers

Reply:
this is the same thing done by the controller runtime zap logger:  https://github.com/kubernetes-sigs/controller-runtime/blob/116a1b831fffe7ccc3c8145306c3e1a3b1b14ffa/pkg/log/zap/zap.go#L42

Reply:
yes but why are we using zapr.NewLogger now?

Reply:
it's how the controller runtime package converts the raw logger to the logr interface so I figured that was the best way to make that conversion

Reply:
The parameters to NewRaw and zap.New are the same. We don't need a raw logger

Reply:
ah, thanks for catching that! I just saw the package was doing it and figured it was necessary but I see what you mean. fixing now